dist/_include-media.scss

```diff
@@ -235,6 +235,7 @@ $im-no-media-expressions: ('screen', 'portrait', 'landscape') !default;
       $prefix: get-expression-prefix($operator);
       $value: get-expression-value($condition, $operator);
 
+      // scss-lint:disable SpaceAroundOperator
```

Comment:
scss-lint complains...

```bash
[23:49:54] Using gulpfile ~/Desktop/Dev/include-media/Gulpfile.js
[23:49:54] Starting 'build'...
[23:49:54] Finished 'build' after 414 ms
[23:49:54] Starting 'test'...
[23:49:54] Starting 'test:libsass'...
[23:49:54] Starting 'test:rubysass'...
[23:49:54] Finished 'test' after 303 ms
WARNING: ✔ All 25 tests from `parse-expression` passed successfully.
	on line 145 of tests/helpers/_SassyTester.scss
	from line 27 of stdin

WARNING: ✔ All 78 tests from `to-number` passed successfully.
	on line 145 of tests/helpers/_SassyTester.scss
	from line 19 of stdin

[23:49:54] Finished 'test:libsass' after 340 ms
[23:49:55] directory ./
[23:49:55] WARNING: ✔ All 25 tests from `parse-expression` passed successfully.
         on line 145 of tests/helpers/_SassyTester.scss, in `run'
         from line 27 of ./tests/parse-expression.scss
[23:49:55] write ./parse-expression.css
[23:49:55] WARNING: ✔ All 78 tests from `to-number` passed successfully.
         on line 145 of tests/helpers/_SassyTester.scss, in `run'
         from line 19 of ./tests/to-number.scss
[23:49:55] write ./to-number.css
[23:49:55] Finished 'test:rubysass' after 797 ms
DEPRECATION WARNING on line 104, column 16 of ./tests/helpers/_SassyTester.scss:
Unescaped multiline strings are deprecated and will be removed in a future version of Sass.
To include a newline in a string, use "\a" or "\a " as in CSS.

./dist/_include-media.scss:238 [W] SpaceAroundOperator: `$prefix == 'max' and $value <= $no-media-breakpoint-value) or ($prefix == 'min' and $value > $no-media-breakpoint-value) ` should be written with a single space on each side of the operator: `$prefix == 'max' and $value <= $no-media-breakpoint-value) or $prefix == 'min' and $value > $no-media-breakpoint-value`
./src/helpers/_no-media.scss:21 [W] SpaceAroundOperator: `$prefix == 'max' and $value <= $no-media-breakpoint-value) or ($prefix == 'min' and $value > $no-media-breakpoint-value) ` should be written with a single space on each side of the operator: `$prefix == 'max' and $value <= $no-media-breakpoint-value) or $prefix == 'min' and $value > $no-media-breakpoint-value`
pre-commit: 
pre-commit: We've failed to pass the specified git pre-commit hooks as the `lint`
pre-commit: hook returned an exit code (1). If you're feeling adventurous you can
pre-commit: skip the git pre-commit hooks by adding the following flags to your commit:
pre-commit: 
pre-commit:   git commit -n (or --no-verify)
pre-commit: 
pre-commit: This is ill-advised since the commit is broken.
pre-commit: 
```